networking/network_observability/network-observability-overview.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id='network-observability-overview']
+= About Network Observability
+include::_attributes/common-attributes.adoc[]
+:context: network-observability-overview
+
+toc::[]
+
+Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters. The NetObserv eBPF agent uses eBPF technology to create network flows based on the IPFIX format. Alternatively, you can export network flows if you are using OVN Kubernetes, but this is not supported.  The network flow data is saved to an object store. You can access the network flow through Loki by using Web console or Grafana
```

Comment:
I think this section is confusing and needs to be rewritten.  We should really call out the value here.   I think it's confusing to talk about eBPF and IPFIX in the same sentence.
I also think the sentence starting "Alternatively" doesn't add much value -- why would we say you can do something but it's not supported -- in the first thing customers see.  
I also think we should probably not mention Grafana.  In this section we should say that Netflow information is available in the OpenShift Console.

Reply:
Per discussion with @mcurry-rh it is essential for us to highlight eBPF and how Network Observability Operator is using it. Thanks!

Reply:
Addressed